integ-test/src/test/java/org/opensearch/sql/legacy/ExplainIT.java

```diff
@@ -7,12 +7,7 @@
 
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
-import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_ACCOUNT;
-import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_DOG;
-import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_LOCATION;
-import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_NESTED_TYPE;
-import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_PEOPLE;
-import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_PHRASE;
+import static org.opensearch.sql.legacy.TestsConstants.*;
```

Comment:
nit: use constants instead of `*`, this change might be introduced by IDE

Reply:
Sure, let me double check. Thanks!

Reply:
Addressed in https://github.com/opensearch-project/sql/pull/3337/commits/84a2fffe7bbf78c738d70f0e430c4fdf49f567f8.